examples/specs/concat_population_pyramid.vl.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A population pyramid for the US in 2000.",
+  "data": { "url": "data/population.json"},
+  "transform": [
+    {"filter": "datum.year == 2000"},
+    {"calculate": "datum.sex == 2 ? 'Female' : 'Male'", "as": "gender"}
+  ],
+  "hconcat": [{
+    "transform": [{
+      "filter": {"field": "gender", "equal": "Female"}
+    }],
+    "title": "Female",
+    "mark": "bar",
+    "encoding": {
+      "y": {
+        "field": "age", "type": "ordinal",
+        "axis": null
+      },
+      "x": {
+        "aggregate": "sum", "field": "people", "type": "quantitative",
+        "axis": {"title": "population"},
+        "sort": "descending"
+      },
+      "color": {
+        "field": "gender", "type": "nominal",
+        "scale": {"range": ["#e377c2","#1f77b4"]},
+        "legend": null
+      }
```

Comment:
I think the reason is that we want to use the same color scale. However, you are right that it would be nice if you only had to specify the color you actually want.